rpc/documentation/api.md

```diff
@@ -635,12 +639,53 @@ search for transactions involving the private key's associated payment address.
 
 **Expected errors:**
 
+- `InvalidArgument`: The private passphrase is incorrect.
+
 - `InvalidArgument`: The private key WIF string is not a valid WIF encoding.
 
+- `InvalidArgument`: A rescan height was specified, but the rescan option was 
+  not set.
+
+- `InvalidArgument`: A negative rescan height was passed.
+
 - `Aborted`: The wallet database is closed.
 
+- `NotFound`: The account does not exist.
+
+**Stability:** Unstable
+
+___
+
+#### `ImportScript`
+
+The `ImportScript` method imports a script into the wallet.  A rescan may 
+optionally be started to search for transactions involving the script, either 
+as an output or in a P2SH input.
+
+**Request:** `ImportScriptRequest`
```

Comment:
Also just noticed that this is missing the account.  We should add it to the message and error whenever a script is imported into anything other than the imported account.  Later on we will enable both script and private key imports for other accounts as well.